drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -1130,93 +1142,58 @@
 
 /*** Assessment window notification ***/
 .assessment-notification {
-    display: table;
-    width: 100%;
-    background-color: #1d5280;
+    background-color: white;
     position: absolute;
-    left: 0;
-    bottom: -60px;
-    right: 0;
-    height: 230px;
-    color: #ffffff;
-    box-shadow: 0 -2px 10px #999;
-    z-index: 10;
-}
-
-.assessment-notification > div{
-    display: table-cell;
-    vertical-align: middle;
-    height: 230px;
+    left: 50%;
+    top: 50%;
+    color: #1d5280;
+    box-shadow: 0 0 15px #333333;
+    z-index: 12;
+    transform: translate(-50%, -50%);
+    padding: 20px;
     text-align: center;
+    width: 330px;
+    height: 260px;
+    border-radius: 3px;
 }
 
-.assessment-notification > .icon{
-    width: 210px;
-}
-
-.assessment-notification > .icon .fa{
-    width: 52px;
-    height: 52px;
-    line-height: 52px;
-    background-color: #ffffff;
+.assessment-notification .header-icon{
+    width: 40px;
+    height: 40px;
+    margin: 10px auto;
+    text-align: center;
+    background-color: #f4921e;
```

Comment:
I think colour to use for LMS here should be #edbd3c. This specific colour can go to the theme.

Reply:
Done